index.html

```diff
@@ -0,0 +1,74 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title> CLARENCE PORTFOLIO</title>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap" rel="stylesheet">
+</head>
+
+
+<body>
+  
+  <header class="header" id="header">
+    <a href="#">CLARENCE</a>
+    <nav>
+      <span class="open" id="menu"></span>
+       <img src="images/Icon - Menu.png" alt="menu bar" class="icon-container">
+      
+    </nav>
+  </header>
+  
+  <!---Main---->
+
+  <main class="main">
+      
+    
+          <div class="home-page">
+              <div>
+                  <h2 class="home_header">Hey There.</h2>
+                  <h3 class="home_header">I'm Clarence Atim</h3>
+              </div>
+
+              <div class="home-description">
+                  <h5 class="intro-heading">I'm a Software Developer</h5>
+
+                  <p class="subject">
+                      I can help you build a product, feature or website Look through some
+                      of my work and experiece! If you like what you see and have a
+                      project you need coded, don't hesitate to contact me.
+                  </p>
+              </div>
+          </div>
```

Comment:
Kindly ensure that the headline section is as given by the design. Adjust the margin of the main title from the toolbar. Also adjust the font of the main title to match the design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/235295619-95553e03-2839-4075-9b18-61aad0432684.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/235295626-2f96cd11-b01c-4b6e-a4a7-afa0a484e0ed.png)